apps/omg/lib/omg/recorder.ex

```diff
@@ -53,11 +51,11 @@ defmodule OMG.Recorder do
   end
 
   def handle_info(:gather, state) do
-    # invoke the reporter function and pass the key and value (invoke the fn)
-    _ =
-      state.reporter.(state.key, Process.info(state.parent, :message_queue_len) |> elem(1),
-        tags: [inspect(%{node: state.node})]
-      )
+    :telemetry.execute(
+      OMG.Utils.Metrics.to_event_name(state.name),
+      %{message_queue_len: Process.info(state.parent, :message_queue_len) |> elem(1)},
```

Comment:
nitpick: how about just `message_queue_size` if we want to save on characters? Avoids needing to truncate a word.

Or alternatively, just spell out  it as `message_queue_length` since we use that for our other metrics and keeps it standardized.

Reply:
you're right, I should change the earlier name

Reply:
https://hexdocs.pm/elixir/Process.html#info/1 -> http://erlang.org/doc/man/erlang.html (https://github.com/erlang/otp/blob/master/erts/doc/src/erlang.xml#L5180-L5201)
Here I wanted to be consistent with the name used in erlang, to easier find alternatives that we can also get from Process.info.

But if you prefer a different name it's no problem, I could change it.